src/main/kotlin/eu/europa/ec/eudi/openid4vp/internal/request/ClientMetaDataValidator.kt

```
@@ -63,29 +61,14 @@ internal class ClientMetaDataValidator(private val httpClient: HttpClient) {
             throw RequestValidationError.InvalidClientMetaData("Invalid vp_format").asException()
         }
 
-    private suspend fun jwkSet(clientMetadata: UnvalidatedClientMetaData): JWKSet {
-        val jwks = clientMetadata.jwks
-        val jwksUri = clientMetadata.jwksUri
-
+    private fun jwkSet(clientMetadata: UnvalidatedClientMetaData): JWKSet {
```

Irrelevant to the removal of `jwks_uri`

The return type should be nullable, given that the caller knows/decides whether the `jwks` is required or not.

Please raise `MissingClientMetadataJwks` from the caller site